readthedocs/core/management/commands/reindex_elasticsearch.py

```diff
@@ -41,10 +45,8 @@ def handle(self, *args, **options):
             except:
                 # This will happen on prod
                 commit = None
+
             try:
-                page_list = parse_json.process_all_json_files(version, build_dir=False)
-                index_search_request(
-                    version=version, page_list=page_list, commit=commit,
-                    project_scale=0, page_scale=0, section=False, delete=False)
```

Comment:
I believe the delete=False is needed here because we don't have a `commit` on these indexes, so they will be deleted instantly on index. The HTML files don't keep track of their commit in a sane way (though I believe we could probably parse it out of the HTML), and that's how we do the tracking around staleness of files in the index.